### DISCUSSION TITLE
some more translations DE

### DISCUSSION THREAD

**Discussion by HurryJupp** [aclap-dev/video-downloadhelper#873] [Language & Translation]
[Translations.de.txt](https://github.com/user-attachments/files/15831599/Translations.de.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
